docs/email.md

```diff
@@ -0,0 +1,17 @@
+# Emails
+
+## Local Testing
+
+We use the gem [Letter Opener Web](https://github.com/fgrehm/letter_opener_web) to deliver emails in development for testing purposes. It provides a web interface based on the original [Letter Opener](https://github.com/ryanb/letter_opener) gem. All delivered emails can be accessed at `localhost:3000/letter_opener`.
```

Comment:
another paragraph here to say what they would do to change it from the web interface to just opening in a new tab on email delivery. show code example to change.